pkg/stub/handler.go

```diff
@@ -40,6 +40,9 @@ const (
 	x86OKDContentRootDir   = "/opt/openshift/operator/okd-x86_64"
 	ppc64OCPContentRootDir = "/opt/openshift/operator/ocp-ppc64le"
 	installtypekey         = "installtype"
+	regkey                 = "reg"
```

Comment:
let's do better with these names, we're going to be living with them for a long time.

Reply:
attempt at better names part of next push